docs/i18n-docs.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Automation tools for i18n and translate
+authors:
+- "@chouchongYHMing" # Authors' github accounts here.
+reviewers:
+- TBD
+approvers:
+- "@warjiang"
+
+creation-date: 2024-08-19
+
+---
+
+# Automation tools for i18n and translate
+
+## Summary
+
+This is an out-of-the-box automated i18n tool designed for the Karmada dashboard, developed primarily from a Chinese environment perspective. It implements the following features:
+
+1. Automatically configures i18n key-value pairs in the form of {key:value}.
+2. Automatically translates all i18n key-value pairs in a directory and adds them to the existing i18n repository.
+
+<!--
+这是一个为Karmada dashboard设计的开箱即用的自动化i18n工具，基于以中文环境开发的主视角，它实现了以下功能：
+1. 自动配置i18n键值对，并且为{key:value}形式。
+2. 自动翻译整个目录下的i18n键值对，并且添加到原有的i18n仓库中。
+-->
+
+## Motivation
+
+The global digital landscape demands applications that cater seamlessly to diverse linguistic audiences. Recognizing this need, we embarked on creating an automated tool designed specifically for the Karmada dashboard, which simplifies the process of internationalizing applications by automatically detecting and converting Chinese text within the code into internationalized function calls. This tool not only enhances the accessibility and usability of applications across different language-speaking regions but also streamlines the development process by integrating internationalization early in the software development lifecycle. 
```

Comment:
the motivation should focus on the i18n automation tools instead of importance of i18n